src/cmd/services/m3coordinator/ingest/carbon/ingest.go

```diff
@@ -41,13 +41,20 @@ import (
 	"github.com/m3db/m3/src/query/ts"
 	"github.com/m3db/m3x/instrument"
 	"github.com/m3db/m3x/log"
+	"github.com/m3db/m3x/pool"
 	m3xserver "github.com/m3db/m3x/server"
 	xsync "github.com/m3db/m3x/sync"
 	xtime "github.com/m3db/m3x/time"
 
 	"github.com/uber-go/tally"
 )
 
+const (
+	maxResourcePoolNameSize = 200
```

Comment:
Maybe bump this to 256 for "power of 2-ness".  Also IDs can get long in graphite land, heh.

Reply:
should I go higher than 256?